tools/gems.md

```diff
@@ -0,0 +1,36 @@
+# Publishing gems with GitHub Packages
+
+## Versions
+
+To release a new version run `gem bump`, which will update the version and create a version commit.
```

Comment:
Do we need to install this?
```
InfirmaryEngine main % gem bump         
ERROR:  While executing gem ... (Gem::UnknownCommandError)
    Unknown command bump
InfirmaryEngine main % gem -v
3.4.21
```

Reply:
I think it would be nice @tute! I think it would be good to replace all our other gems publishing info in their READMEs to point to this new guide.

With that would be good to add the `gem` gem (har har gem gem) to them. I found that `gem` when working on the `rubocop-buoy` gem and browsing the Github pages.

Reply:
I am confused as to the need to add a `gem "gem"` _gem_ declaration 🤣 at this point.

Reply:
Oh wait, sorry this comes from an extension of `gem`. It's specifically the `gem-release` gem.

https://github.com/BuoySoftware/rubocop-buoy/blob/main/Gemfile#L8
https://github.com/svenfuchs/gem-release

Reply:
@tute @rayfaddis I added the `gem-release` requirement to the document in 94cae5eae0750f30f0caf2d034a2c5936b0022ba.

Reply:
@arzezak do we need that in the publishing instructions or do we just need to make sure it's added to all our gems Gemfiles instead?

Reply:
That's a good question. Maybe removing the instructions on how to do it but leaving a reminder that we assume it to be there would be better?